src/Screens/Cart/styles.ts

```diff
@@ -1,10 +1,11 @@
 import { StyleSheet } from "react-native";
+import { colors } from "../../assets/arrays/theme";
```

Comment:
Use Absolute paths if possible,
Also, colors will go inside the theme directory ( direct child of src directory ).